demo/webcam_api_demo.py

```diff
@@ -51,6 +52,10 @@ def set_device(cfg: Config, device: str):
 
 
 def run():
+
+    warnings.warn('The Webcam API will be deprecated in future. ',
```

Comment:
Please add a reference to the inferencer-based webcam demo.

Reply:
In fact, the inferencer utilizing webcam input encounters a similar issue as the webcam API, where the visualization process is too slow to render keypoints in real time. As a result, users might have a less satisfactory experience with the inferencer compared to the updated webcam API.

Reply:
Ok, I think this PR can be merged as it is.